<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1347</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="DelSelect"/>
      </item>
      <item>
       <widget class="QPushButton" name="DelBtn">
        <property name="text">
         <string>Удалить аккаунт</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="AddAccounts">
      <property name="text">
       <string>Добавить аккаунты</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QTableWidget" name="table">
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Запуск</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Прокси</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Аккаунт</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Лайк</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Комментарий</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Ссылка</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Текст коментария</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Задержка</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Статус выполнения задачи</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="Publics">
      <property name="text">
       <string>Паблики для автолайкинга</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1347</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
